### ISSUE TITLE
Proto comments in reflection

### ISSUE THREAD

**Issue by Paxa** [grpc/grpc#22680]
What do you think about adding code comments to reflection message for documentation purpose?

Then it will be possible to build documentation like this https://github.com/pseudomuto/protoc-gen-doc dynamically from server, and combine with UI like this https://github.com/gusaul/grpcox#demo

It can be like openapi for grpc

May be can make comments more structured, something like javadoc, can separate request example and response examples

**Comment by kgb-financial-com** [comment]
It's a pity that this got auto-closed, seems like a really good suggestion to me.

**Comment by AlexMihalev** [comment]
Yep, in contradiction with GraphQL or OpenAPI where documentation plays a significant role, GRPC looks like don't bother about documentation so much.

**Comment by shalako** [comment]
I'm looking for this now. Our source code repos are unfortunately not available to our API consumers, and the proto comments provide crucial documentation. Was really hoping that reflection could be used to expose these comments.
